### PR TITLE
docs cleanup

### PR DESCRIPTION
**Pull request by herbig**
## Description

Cleaning up typos and missing definitions for docs.

## Notes

Some more TODOs to discuss with @tbwebb22

### CODE REVIEW

File: contracts/BaseFreezeVoting.sol [hunk 1]
```diff
@@ -23,12 +23,25 @@ import "./interfaces/IBaseFreezeVoting.sol";
  */
 abstract contract BaseFreezeVoting is FactoryFriendly, IBaseFreezeVoting {
 
-    uint256 public freezeVotesThreshold; // number of freeze votes required to activate a freeze
-    uint256 public freezeProposalCreatedBlock; // block number the freeze proposal was created at
-    uint256 public freezeProposalVoteCount; // number of accrued freeze votes
-    uint256 public freezeProposalPeriod; // number of blocks a freeze proposal has to succeed
-    uint256 public freezePeriod; // number of blocks a freeze lasts, from time of freeze proposal creation
+    /** Number of freeze votes required to activate a freeze. */
+    uint256 public freezeVotesThreshold;
```
Comment (herbig) [inline contracts/BaseFreezeVoting.sol:27]: moved these to natspec so they'll appear in the docs

File: contracts/azorius/Azorius.sol [hunk 2]
```diff
@@ -6,7 +6,7 @@ import "./interfaces/IBaseStrategy.sol";
 import "./interfaces/IAzorius.sol";
 
 /**
- * @title Azorius Protocol - a Safe module which allows for composable governance.
```
Comment (herbig) [inline contracts/azorius/Azorius.sol:9]: The `@title` tag broke the whole doc from appearing, so I just removed them.

File: contracts/azorius/Azorius.sol [hunk 3]
```diff
@@ -81,14 +91,17 @@ contract Azorius is Module, IAzorius {
     error InvalidTxs();
     error InvalidArrayLengths();
 
+    /**
+     * Initial setup of the Azorius instance.
+     */
     function setUp(bytes memory initParams) public override initializer {
         (
-            address _owner,
+            address _owner,                 // TODO what should these say?
```
Comment (herbig) [inline contracts/azorius/Azorius.sol:99]: @tbwebb22 what are these three setup parameters exactly? (`owner`, `avatar`, `target`)

File: contracts/azorius/LinearERC20Voting.sol [hunk 4]
```diff
@@ -105,7 +105,7 @@ contract LinearERC20Voting is BaseStrategy, BaseQuorumPercent {
      * @param _proposalId id of the Proposal to vote on
      * @param _voteType Proposal support as defined in VoteType (NO, YES, ABSTAIN)
      */
-    function vote(uint256 _proposalId, uint8 _voteType, bytes memory) external {
+    function vote(uint256 _proposalId, uint8 _voteType, bytes memory) external { // TODO what's this unused parameter???
```
Comment (herbig) [inline contracts/azorius/LinearERC20Voting.sol:108]: @tbwebb22 this `bytes memory` is unnamed and seemingly unused, pretty sure we can remove it?

File: README.md [hunk 5]
```diff
@@ -11,6 +11,9 @@ All voting details are delegated to BaseStrategy implementations, of which an Az
 
 Azorius was forked from and heavily based on the [Usul](https://github.com/SekerDAO/Usul) module, by [SekerDAO](https://github.com/SekerDAO).
 
+## Contract Documentation
+[NatSpec](https://docs.soliditylang.org/en/v0.8.17/natspec-format.html) documentation for Azorius Protocol contracts [are available here](https://github.com/decent-dao/fractal-contracts/tree/main/docs).
```
Comment (herbig) [inline README.md:15]: this link will be dead until we merge into main

File: docs/MultisigFreezeGuard.md [hunk 6]
```diff
@@ -2,41 +2,48 @@
 
 ## MultisigFreezeGuard
 
-A Safe Transaction Guard contract that prevents an multisig (Safe) subDAO from executing transactions 
-if it has been frozen by its parentDAO.
-
-see https://docs.safe.global/learn/safe-core/safe-core-protocol/guards
+Implementation of IMultisigFreezeGuard. See the IMultisigFreezeGuard doc more for details.
```
[the remark this answers is not in the document]
Comment (herbig) [reply docs/MultisigFreezeGuard.md:5]: yeah, all of these .md files are generated from the NatSpec comments in the Solidity code, and Natspec doesn't have a way to add links to other files..

File: docs/MultisigFreezeGuard.md [hunk 7]
```diff
@@ -2,41 +2,48 @@
...
 
+Execution period (in blocks).
+
 ### freezeVoting
 
 ```solidity
 contract IBaseFreezeVoting freezeVoting
 ```
 
+Reference to the IBaseFreezeVoting implementation that determines whether the Safe is frozen.
```
[the remark this answers is not in the document]
Comment (herbig) [reply docs/MultisigFreezeGuard.md:29]: same here, these are generated docs

File: docs/azorius/interfaces/IAzorius.md [hunk 8]
```diff
@@ -53,23 +62,23 @@ created using any of the currently enabled strategies.
...
+| _strategy | address | contract address of the BaseStrategy to be enabled |
 
 ### disableStrategy
 
 ```solidity
 function disableStrategy(address _prevStrategy, address _strategy) external
 ```
 
-Disables a previously enabled BaseStrategy implementation for new proposal.
+Disables a previously enabled BaseStrategy implementation for new proposals.
```
[the remark this answers is not in the document]
Comment (herbig) [reply docs/azorius/interfaces/IAzorius.md:73]: my general rule was to use punctuation and full sentences in the documentation comments, but not in `@param` and `@returns` tags

File: docs/MultisigFreezeGuard.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (herbig) [reply docs/MultisigFreezeGuard.md:5]: oh shoot, actually I can just use regular markdown links...

now I need to go back and do that throughout my comments, ha 

I'll do links in a followup PR

File: contracts/azorius/Azorius.sol (hunk 3 above)
Comment (tbwebb22) [reply contracts/azorius/Azorius.sol]: So `owner` is the address that can call all of the `onlyOwner` functions. 
The `target` is the contract that transactions are first passed to. 
And the `avatar` is the contract that the transactions will ultimately be executed by. 

In our case, these three will probably all always be the same - the Gnosis Safe. I guess a case when avatar and target could be different, is for some reason if transactions need to be passed through another module, or some other contract. But I'm not really sure of when or why that would be needed.

**Pull request merged**